pathway/static/js/pathway/controllers/find_patient.js

```diff
@@ -35,7 +35,7 @@ controllers.controller('FindPatientCtrl',
     vm.new_patient = function(result){
         vm.state = 'editing_demographics';
         vm.demographics.hospital_number = vm.hospital_number;
-        focus('input[name="patient_demographics[0]_name"]');
+        focus('input[name="patient_demographics[0]_first_name"]');
     };
```

Comment:
Is this not going to blow up with the same focusProvider error we saw somewhere yesterday?